packages/server/utils/Logger.ts

```diff
@@ -0,0 +1,35 @@
+import tracer from 'dd-trace'
+import formats from 'dd-trace/ext/formats'
+import util from 'util'
+
+type LogLevel = 'error' | 'warn' | 'info' | 'debug'
+const LogFun = {
+  error: console.error,
+  warn: console.warn,
+  info: console.info,
+  debug: console.debug
+} satisfies Record<LogLevel, Console[keyof Console]>
+
+function trace(level: LogLevel, message: any, ...optionalParameters: any[]) {
+  if (!process.env.DD_LOGS_INJECTION) {
```

Comment:
-1 env vars are strings, should we instead say `!== 'true'`?

Reply:
Yes, I wouldn't have checked an env if there wasn't already one defined and set in our environments.